src/plugins/intel_cpu/src/cpu_memory.cpp

```diff
@@ -35,17 +35,24 @@ namespace {
 
     void transferData(const IMemory& src, const IMemory& dst, bool ftz) {
         node::Reorder::reorderData(src, dst);
+        try {
+            auto src_data_type = DnnlExtensionUtils::IEPrecisionToDataType(src.getDesc().getPrecision());
+            auto dst_data_type = DnnlExtensionUtils::IEPrecisionToDataType(dst.getDesc().getPrecision());
+            if (src_data_type != memory::data_type::f32 || dst_data_type == memory::data_type::bf16) {
+                return;
+            }
+        } catch (ov::Exception&) {
+            return;
+        }
```

Comment:
This method should be implemented little bit differently. There we have a bunch of checks to make a decision on whether it is allowed to perform flushing subnormals to zero. These checks use different oneDNN entities:
```cpp
src.getDataType() == memory::data_type::f32
dst.getDataType() != memory::data_type::bf16
``` 
Should be simply replaced with
```cpp
srd.getDesc().getPrecision() == Precision::FP32
dst.getDesc().getPrecision() == Precision::BF16
```
The checks below:
```cpp
!wrapper.is_rnn_packed_desc()
!wrapper.is_wino_desc()
```
make sense only for Dnnl Type memory descriptors, thus should be extracted and performed only if the dst memory has dnnl memory desc:
```cpp
if (!ftz) {
    return;
}
if (srd.getDesc().getPrecision() != Precision::FP32 ||
    dst.getDesc().getPrecision() == Precision::BF16) {
    return;
}
size_t offset = 0;
if (dst.getDesc().getType() & Type::Dnnl) {
    // here we can safely cast to DnnlMemoryDesc
    auto dnnl_desc =dst.getMemoryDescAs<DnnlMemoryDesc>();
    auto desc = dnnl_desc ->getDnnlDesc();
    dnnl::impl::memory_desc_wrapper wrapper(desc.get());
    offset = wrapper.offset0();
    if (wrapper.is_wino_desc() || wrapper.is_rnn_packed_desc()) {
        return;
    }
}
// actual FTZ
auto *memData = static_cast<float *>(dst.getData());
memData += offset;
setSubnormalsToZero(memData, dst.getSize() / sizeof(float));
```

Reply:
If  ` (dst.getDesc().getType() & MemoryDescType::Dnnl)==false`  it still do FTZ?

Reply:
In that case we don't need to check for `is_wino_desc` and `is_rnn_packed_desc`, so we simply skip those checks.